package.json

```diff
@@ -22,13 +14,13 @@
     "buildProdNode": "NODE_ENV=true ENV=prod uglify=true rollup -c",
     "buildForNpm": "NPM=true rollup -c",
     "bundle-size-visual": "NODE_ENV=true ENV=prod PROD_DEBUG=true uglify=true visualizer=true rollup -c",
-    "size-build": "npm run buildProdBrowser",
-    "size": "npm run size-build && size-limit",
+    "size": "npm run buildProdBrowser && npm run buildAllIntegrations && npm run buildForNpm && size-limit",
     "changelog": "auto-changelog -p -t keepachangelog -u true -l false --sort-commits date-desc ",
     "buildDevIntegration": "NODE_ENV=false uglify=false INTG_NAME=AdobeAnalytics rollup --config rollup.intgConfig.js",
     "buildDevIntegrationCLI": "NODE_ENV=false uglify=false INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js",
     "buildProdIntegration": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=AdobeAnalytics rollup --config rollup.intgConfig.js",
-    "buildProdIntegrationCLI": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js"
+    "buildProdIntegrationCLI": "NODE_ENV=false ENV=prod uglify=true INTG_NAME=$npm_config_intg rollup --config rollup.intgConfig.js",
```

Comment:
Add `PROD_DEBUG=true` for these commands.

Reply:
This is already taken care in this PR,
https://github.com/rudderlabs/rudder-sdk-js/pull/537/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519